### DISCUSSION TITLE
Please, help to run elFinder! Cannot even try it...

### DISCUSSION THREAD

**Discussion by azlk** [Studio-42/elFinder#3352] [Q&A]
It seems to me I have fulfilled all the conditions for installation:
1.Download and unzip one of the releases below to your PHP server
2.Rename /php/connector.minimal.php-dist to /php/connector.minimal.php
3.Load /elfinder.html in your browser to run elFinder
Also, if I get it right, to load elfinder.html in browser I need to create a virtual host file for web server? I created one for nginx:
```
server {
	listen 80;
	root /home/elFinder/elFinder-2.1.58;
	index elfinder.html;
	location / {
		try_files $uri $uri/ =404;
	}
}
```
Actually I need to list files in **/var/www/vbr**, so I doubt is this vhost configuration is correct?
I tried also with "root /var/www/vbr" but then I get "404 Not found" (no surprise).
In browser I get "Некорректный ответ сервера. Данные не в формате JSON"
```
nginx version: nginx/1.18.0 (Ubuntu)
php -v
PHP 7.4.3 (cli) (built: Oct  6 2020 15:47:56) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.3, Copyright (c), by Zend Technologies
libjs-jquery and libjs-jquery-ui installed.
![elfinder1](https://user-images.githubusercontent.com/4054635/121566275-010b0700-ca26-11eb-897b-8e24143e4130.png)

```
Actually I hardly understand, where exactly should I write the path to the files - in which file??? 
1. In file connector_minimal.php
```
	'roots' => array(
		// Items volume
		array(
		'driver'  => 'LocalFileSystem', // driver for accessing file system (REQUIRED)
here?-->        'path'   => '/var/www/vbr/', // path to files (REQUIRED)
or here? which files were meant??? --> 'URL' => dirname($_SERVER['PHP_SELF']) . '../files', // URL to files (REQUIRED)
What path would be correct here for my case?
```
_or_
2. In elfinder.html
**var path = ' '** (line 50)
??
And **where exactly** can I write lang = ru? I could not find a clear description in Wiki, sorry...
_Please, help me to clear those questions, I am totally new to js and my questions may look dumb, I understand, but I liked elFinder very much but couldn't even look at it, how it would work on my server._

**Comment by nao-pon** [comment]
@azlk When asking a question you don't understand, it's important where you ask. This time you're stopping at building a web server first, so the place you should ask is the community for building a web server.

By doing so, I think you will get a clearer answer.

If you can build a web server that can run PHP correctly and the elFinder installation doesn't work, you can get the right answer by explaining what to do and what doesn't work. ..
